src/pipeline/node/Sync.cpp

```
@@ -1,51 +1,91 @@
 #include <chrono>
+#include <stdexcept>
 
 #include "depthai/pipeline/datatype/MessageGroup.hpp"
 #include "pipeline/ThreadedNodeImpl.hpp"
 
 namespace dai {
 namespace node {
 
-void Sync::setSyncThreshold(std::chrono::nanoseconds syncThreshold) {
-    properties.syncThresholdNs = syncThreshold.count();
+namespace {
```

Cppcheck (2.20.0)

[information] Include file

(missingIncludeSystem)